src/css/controls/imports/autostartmute.less

```diff
@@ -2,7 +2,7 @@
 
 .jw-autostart-mute {
     position: absolute;
-    bottom: 0.5em;
+    bottom: 4em;
```

Comment:
What's this for (please cover in the PR description)?

Reply:
@robwalch Added it. I adjusted the position so the mute icon so the control bar is fully visible while the user is tapping the player while the mute button is displayed (it covers up the fullscreen icon)